modules/ROOT/pages/_partials/sync/provisioning.adoc

```diff
@@ -0,0 +1,62 @@
+
+It's neccessary to wrap your Sync Server application into Docker container and publish your container to a Docker registry. 
+The easiest way is to use a public one such as link:https://hub.docker.com/[Dockerhub].
+
+= Preparing for deployment
```

Comment:
Maybe "Building and Publishing the container".  It is much more clear about what we're doing than our intention.

Reply:
ok